setup/dist/action.yml

```diff
@@ -1,48 +0,0 @@
-name: 'Setup Haskell'
```

Comment:
does this file need to be there?

Reply:
[https://github.com/haskell/actions/blob/745062a754c3c4b70b87cb93937ad443096cc94d/setup/src/opts.ts#L37](`outputs.ts`) actually reads the `action.yml` file in order to automatically grab all the inputs and prevent it from not being the source of truth regarding how inputs are defined and working.